config/runtime/tests/fluster-chromeos.jinja2

```diff
@@ -66,4 +74,5 @@
               -i /home/cros/.ssh/id_rsa
               root@$(lava-target-ip)
               poweroff && sleep 30 || true
-            - python3 /usr/bin/fluster_parser.py --results
```

Comment:
You are calling this commands after issuing poweroff

Reply:
@nuclearcat poweroff is issued on the DUT, this command runs on lava/docker

Reply:
Ah ok then, i guess it should not disrupt anything.